src/device/usbd.c

```diff
@@ -611,9 +586,10 @@ static bool process_control_request(uint8_t rhport, tusb_control_request_t const
           case TUSB_REQ_SET_INTERFACE:
           {
             uint8_t const alternate = (uint8_t) p_request->wValue;
+            (void) alternate;
 
             // TODO not support alternate interface yet
-            TU_ASSERT(alternate == 0);
+//            TU_ASSERT(alternate == 0);
```

Comment:
This isn't technically correct, but should be fixed soon with other PR.

Reply:
yeah, it is tempt hack to test out the request. I forgot to revert the change.